client/src/features/maintenance/Maintenance.tsx

```diff
@@ -16,47 +16,41 @@
  * limitations under the License.
  */
 
-/**
- *  renku-ui
- *
- *  Maintenance.js
- *  Maintenance components.
- */
-
 import React from "react";
 import { BrowserRouter as Router, Route } from "react-router-dom";
 import { Button, Row, Col } from "reactstrap";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faWrench } from "@fortawesome/free-solid-svg-icons";
 
-import { MaintenanceNavBar, FooterNavbar } from "./landing";
-import { StatuspageDisplay, isStatusConfigured } from "./statuspage";
+import { FooterNavbar } from "../../landing";
+import { StatuspageDisplay, isStatusConfigured } from "../../statuspage";
 
 interface MaintenanceProps {
   info: string;
 }
 function Maintenance(props: MaintenanceProps) {
   const { info } = props;
 
-  const headerText = "Maintenance";
+  const headerText = "Maintenance ongoing";
   const body =
     info && info !== "true" && info !== "1"
       ? info
       : "Renku is undergoing maintenance. It should be available again soon. Please check back in a little while.";
   return (
     <Router>
-      <div>
-        <Route component={MaintenanceNavBar} />
-        <main role="main" className="container-fluid">
-          <section className="jumbotron-header rounded px-3 px-sm-4 py-3 py-sm-5 text-center mb-3">
-            <h1 className="text-center text-primary">
-              <FontAwesomeIcon icon={faWrench} /> {headerText}{" "}
-              <FontAwesomeIcon icon={faWrench} />
-            </h1>
-            <br />
-            <p className="text-center">{body}</p>
-          </section>
-        </main>
+      <div className="min-vh-100 d-flex flex-column">
+        <div className="flex-grow-1">
+          <main role="main" className="container-fluid">
+            <section className="jumbotron-header rounded px-3 px-sm-4 py-3 py-sm-5 text-center mb-3">
+              <h1 className="text-center text-primary">
+                <FontAwesomeIcon icon={faWrench} /> {headerText}{" "}
+                <FontAwesomeIcon icon={faWrench} />
+              </h1>
+              <br />
+              <p className="text-center">{body}</p>
```

Comment:
No extra `[newline]`:

```suggestion
              <p className="text-center mt-4">{body}</p>
```